### PR TITLE
added priority and wave fees to filing header schema

### PR DESCRIPTION
**Pull request by kialj876**
Signed-off-by: Kial Jinnah <kialj876@gmail.com>

*Issue #:* /bcgov/entity#2793

*Description of changes:*
- added in priority and waveFees to schema for filing header
- tests and linters pass, old filing json will still pass as well

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-schemas license (Apache 2.0).

### CODE REVIEW

File: src/registry_schemas/example_data/schema_data.py [hunk 1]
```diff
@@ -25,7 +25,9 @@
             'certifiedBy': 'full name',
             'email': 'no_one@never.get',
             'filingId': 1,
-            'routingSlipNumber': '123456789'
+            'routingSlipNumber': '123456789',
+            'waveFees': False,
```
Comment (severinbeauvais) [inline src/registry_schemas/example_data/schema_data.py]: Typo: should be "waiveFees".

Please do global search & replace.

File: src/registry_schemas/example_data/schema_data.py [hunk 2]
```diff
@@ -335,7 +337,8 @@
             'email': 'no_one@never.get',
             'date': '2020-02-18',
             'affectedFilings': [101, ],
-            'routingSlipNumber': '123456789'
+            'waveFees': True,
+            'priority': False
```
Comment (severinbeauvais) [inline src/registry_schemas/example_data/schema_data.py]: The UI won't send 'priority' if 'waiveFees' is True, so a better example might be to omit the 'priority' property here.

File: src/registry_schemas/example_data/schema_data.py (hunk 1 above)
Comment (katiemcgoff) [reply src/registry_schemas/example_data/schema_data.py]: I concur.

File: src/registry_schemas/example_data/schema_data.py (hunk 2 above)
Comment (katiemcgoff) [reply src/registry_schemas/example_data/schema_data.py]: I concur, an example where the field is not present at all would be good to illustrate that it's not required.

File: src/registry_schemas/example_data/schema_data.py (hunk 1 above)
Comment (kialj876) [reply src/registry_schemas/example_data/schema_data.py]: done

File: src/registry_schemas/example_data/schema_data.py (hunk 2 above)
Comment (kialj876) [reply src/registry_schemas/example_data/schema_data.py]: done

**Pull request merged**